### PR TITLE
add-labels-for-etcd-monitoring

### PR DESCRIPTION
**Pull request by calvix**
add labels that are needed for Prometheus config controller in order to support etcd monitoring

towards
https://github.com/giantswarm/giantswarm/issues/3248#issuecomment-403785899

### CODE REVIEW

File: service/controller/v13/resource/service/desired.go [hunk 1]
```diff
@@ -22,11 +22,13 @@ func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interf
...
-				"customer":                   key.CustomerID(customObject),
-				"giantswarm.io/cluster":      key.ClusterID(customObject),
-				"giantswarm.io/organization": key.CustomerID(customObject),
+				"app":                          "master",
+				"cluster":                      key.ClusterID(customObject),
+				"customer":                     key.CustomerID(customObject),
+				"giantswarm.io/cluster":        key.ClusterID(customObject),
+				"giantswarm.io/etcd_domain":    key.ClusterEtcdDomain(customObject),
+				"giantswarm.io/organization":   key.CustomerID(customObject),
+				"giantswarm.io/version_bundle": key.VersionBundleVersion(customObject),
```
Comment (xh3b4sd) [inline service/controller/v13/resource/service/desired.go]: Same with key package constants.

**Comment by calvix** [comment]
is Tim happy?

**Comment by xh3b4sd** [comment]
Well, lets ask him. Tim, are you happy though?

**Comment by xh3b4sd** [comment]
get_off_my_lawn_clint

**Comment by xh3b4sd** [comment]
grandma_finds_the_internet

**Comment by xh3b4sd** [comment]
thumbs_up

**Comment by calvix** [comment]
:trollface:

**Pull request merged**